doc/user/truffleruby-additions.md

```diff
@@ -52,6 +52,44 @@ TruffleRuby provides these non-standard methods and classes that provide additio
 
 * `AtomicReference` is marshalable.
 
+### Concurrent Maps
+
+`TruffleRuby::ConcurrentMap` is a key-value data structure, like a `Hash` and using `#hash` and `#eql?` to compare keys and identity to compare values. Unlike `Hash` it is unordered. All methods on `TruffleRuby::ConcurrentMap` are thread-safe but should have higher concurrency than a fully syncronized implementation. It is intended to be used by gems such as Concurrent Ruby - please use via this gem rather than using directly in most cases.
```

Comment:
Could you link `Concurrent Ruby` to https://github.com/ruby-concurrency/concurrent-ruby?
Should probably be `concurrent-ruby` since that's the gem name (in a Gemfile).

> please use via this gem rather than using directly in most cases.

Could you drop ` in most cases`? I think we want a clear and simple message here.

Reply:
Done.

Reply:
Done.